tests/test_gitlab.py

```diff
@@ -33,25 +33,25 @@ def test_is_same_user(self):
         self.assertTrue(Provider.is_same_user(this, that))
 
     @patch("pyup.providers.gitlab.Gitlab")
-    def test_api(self, github_mock):
+    def test_api(self, gitlab_mock):
```

Comment:
:1st_place_medal: you were really paying attention to stuff here. Congrats. :slightly_smiling_face: